### ISSUE TITLE
100g/400g/800g is missing from available speeds

### ISSUE THREAD

**Issue by scotho3** [codeout/vscode-junos#17]
<img width="563" alt="Screen Shot 2022-09-15 at 1 11 41 PM" src="https://user-images.githubusercontent.com/2288647/190468192-f0ab0a09-433d-4ca2-bd12-3f6b0aed51b4.png">

```
set interfaces et-0/0/1 speed 100g
set interfaces et-0/0/1 ether-options 802.3ad ae5
```

I do see quite a few others listed, `40g`, `10g`, etc.

**Comment by codeout** [comment]
Thanks for pointing it out. I'll take a look.

**Comment by codeout** [comment]
Hi @scotho3, just published `v0.2.6` with the fix.

<img width="761" alt="image" src="https://user-images.githubusercontent.com/921696/191082872-4a36c0a6-19cf-45a2-8f39-611a8cd9898c.png">

Please reopen this issue if you see any further problems.
